Model/Gallery.php

```diff
@@ -43,7 +43,7 @@
     /**
      * @var string
      */
-    protected $defaultFormat;
+    protected $defaultFormat = 'reference';
```

Comment:
Don't worry, you don't have anything to do. I was suggesting changing the doctrine mapping to set this as a default on the database level, but it turns out you can't.